ipcc/src/ffi.rs

```diff
@@ -0,0 +1,70 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+// Copyright 2023 Oxide Computer Company
+
+#![allow(non_upper_case_globals)]
+#![allow(non_camel_case_types)]
+#![allow(non_snake_case)]
+
+use std::ffi::{c_char, c_int, c_uint};
+
+/// Opaque libipcc handle
+#[repr(C)]
+pub(crate) struct libipcc_handle_t {
+    _data: [u8; 0],
+    _marker: core::marker::PhantomData<(*mut u8, core::marker::PhantomPinned)>,
+}
+
+pub(crate) const LIBIPCC_ERR_OK: libipcc_err_t = 0;
```

Comment:
nit: I would prefer if there are newlines between the constants, since it helps with readability.

Reply:
Fixed in 77bd065.